test/angleTokens/ConnectAgTokenSideChainMultiBridges.t.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import { MockSafe } from "../mock/MockSafe.sol";
+import { BaseTest } from "../BaseTest.t.sol";
+import "../../scripts/foundry/Constants.s.sol";
+
+contract ConnectAgTokenSideChainMultiBridgesTest is BaseTest {
+    function testScript() external {
+        (Transaction[] memory transactions) = _deserializeJson();
+
+        for (uint256 i = 0; i < transactions.length; i++) {
+            Transaction memory transaction = transactions[i];
+            address to = transaction.to;
+            uint256 operation = transaction.operation;
+            bytes memory payload = transaction.data;
+            uint256 chainId = transaction.chainId;
+
+            address gnosisSafe = _chainToContract(chainId, ContractType.GuardianMultisig);
+
+            vm.selectFork(forkIdentifier[chainId]);
+
+            // Verify that the call will succeed
+            MockSafe mockSafe = new MockSafe();
+            vm.etch(gnosisSafe, address(mockSafe).code);
+            vm.prank(gnosisSafe);
+            (bool success, ) = gnosisSafe.call(abi.encode(address(to), payload, operation, 1e6));
+            if (!success) revert();
+        }
+
+        string memory stableName = vm.envString("STABLE_NAME");
+        uint256 chainId = vm.envUint("CHAIN_ID");
+        string memory json = vm.readFile(JSON_ADDRESSES_PATH);
+        address token = vm.parseJsonAddress(json, ".agToken");
+        address lzToken = vm.parseJsonAddress(json, ".lzAgToken");
+
+        (uint256[] memory chainIds, address[] memory contracts) = _getConnectedChains(stableName);
+        for (uint256 i = 0; i < contracts.length; i++) {
+            if (chainIds[i] == chainId || chainIds[i] == CHAIN_ETHEREUM) {
```

Comment:
is that the special case you were talking about for mainnet?

Reply:
yep indeed

Reply:
as the setTrustedRemote for mainnet isn't done I need to skip it